projects/saas-ui/src/app/on-boarding/add-lead/add-lead.module.ts

```diff
@@ -0,0 +1,10 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+
+import {AddLeadRoutingModule} from './add-lead-routing.module';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, AddLeadRoutingModule],
+})
```

Comment:
why make modules for each component 
when they can be managed under same root module of on-boarding